apps/balancer-tools/src/app/apr/pool/(components)/HistoricalCharts/HistoricalAPR/HistoricalAPRPlot.tsx

```diff
@@ -1,22 +1,11 @@
 "use client";
 
-import { usePathname, useRouter } from "next/navigation";
-import { PlotMouseEvent } from "plotly.js";
-
 import Plot from "#/components/Plot";
 
 export default function HistoricalAPRPlot({ data }: { data: Plotly.Data[] }) {
-  const router = useRouter();
-  const currentPathname = usePathname();
-  function onClickHandler(event: PlotMouseEvent) {
-    const newPath = currentPathname.replace(/\/round\/\d+$/, "");
-    router.push(`${newPath}/round/${event.points[0].x}`);
-  }
-
   return (
-    <div className="flex justify-between border border-blue6 bg-blue3 rounded p-4 cursor-pointer">
+    <div className="flex justify-between bg-blue3 rounded p-4 cursor-pointer z-50">
```

Comment:
This was my take on the onClick issue, but without success

Reply:
what was the issue again?